README.md

```diff
@@ -39,4 +39,4 @@ $ kubectl get no -o json | jq '.items[].metadata.labels'
 
 ```
 
-[documentation]: https://kubernetes-sigs.github.io/node-feature-discovery
+[documentation]: https://kubernetes-sigs.github.io/node-feature-discovery/v0.14
```

Comment:
We don't need the version here. The link will always get redirected to the latest stable version

Reply:
this is part of `hack/prepare-release` ;P